luthier/config/src/config.lua

```diff
@@ -36,16 +37,34 @@ local metricConfig = metricsUtils.build_metric_config(metrics)
 local metricMakerConfig = metricMakerUtils.build_metric_maker_config(metricMakers)
 local physicalTableConfig, logicalTableConfig = tableUtils.build_table_config(tables)
 
+--- write to testResource/tableName
+local function write_table(targetDir, tableName, configTable)
+    parser.save(targetDir .. tableName, configTable)
+    print(targetDir .. " table " .. tableName .. " Updated now.")
```

Comment:
Why not just use `targetDir .. tableName .. " Updated now."`? That way, it'll be much more
obvious to a customer where their is going.

Reply:
done